lib/product-import.coffee

```diff
@@ -253,7 +243,7 @@ class ProductImport
       @logger.error "Error callback has to be a function!"
 
   _handleFulfilledResponse: (res) =>
-    switch res.value().statusCode
+    switch res.value()?.statusCode
```

Comment:
We don't process the returned value of `_handleFulfilledResponse` anywhere in the code but in case of falsy `res.value()` it won't update `@_summary` stats

Reply:
Should it be an error?